docs/en_US/NAS/SearchSpaceZoo.md

```diff
@@ -172,4 +172,65 @@ All supported operations for ENAS macro search are listed below.
     ..  autoclass:: nni.nas.pytorch.search_space_zoo.enas_ops.PoolBranch
     ```
 
-<!-- push -->
+## NAS-Bench-201
+
+NAS Bench 201 defines a unified search space, which is algorithm agnostic. The predefined skeleton consists of a stack of cells that share the same architecture. Every cell contains four nodes and a DAG is formed by connecting edges among them, where the node represents the sum of feature maps and the edge stands for an operation transforming a tensor from the source node to the target node. The predefined candidate operations can be found in [reference](#nas-bench-201-reference).
+
+The search space of NAS Bench 201 is shown below.
+
+![](../../img/NAS_Bench_201.svg)
+
+```eval_rst
+..  autoclass:: nni.nas.pytorch.nas_bench_201.NASBench201Cell
+    :members:
+```
+
+### Example code
+
+[example code](https://github.com/microsoft/nni/tree/master/examples/nas/search_space_zoo/nas_bench_201.py)
+
+```bash
+# for structure searching
+git clone https://github.com/Microsoft/nni.git
+cd nni/examples/nas/search_space_zoo
+python3 nas_bench_201.py
+```
+
+<a name="nas-bench-201-reference"></a>
+
+### Reference
```

Comment:
"Reference" -> "Candidate operators"

Reply:
I think "Reference" is better here because it is consistent with Darts and ENAS doc.

Reply:
so you should update them as well